styles/youtube/catppuccin.user.css

```diff
@@ -926,7 +926,7 @@
     /* Video highlight */
     ytd-rich-item-renderer.ytd-rich-item-renderer-highlight {
         background-color: fade(@accent-color, 10%) !important;
-        box-shadow: fade(@accent-color, 10%) 0px 0px 0px 10px !important;
```

Comment:
Hold up what?? The linter is asking you to change them to `0` from `0px`, not to change the box shadow entirely?

Reply:
Oh I thought it was saying you couldn't set it to 0.

Reply:
Please ask if you are confused. `deno task lint:fix` would have fixed it for you correctly, and you can reference the documentation: https://stylelint.io/user-guide/rules/length-zero-no-unit/.

Reply:
Thanks, I'll use that the next time.